src/js/dp/search.js

```diff
@@ -18,31 +9,132 @@ const fetchHtml = async (url) => {
   return response && await response.text();
 };
 
-const filters = document.querySelectorAll(".child-filter input");
-let strParams = window.location.search;
+const replaceElementIEPollyfill = (el1, el2) => {
+  el1.insertAdjacentElement('beforebegin', el2);
+  el1.parentElement.removeChild(el1); // absolutely ridiculous
```

Comment:
if "ridiculous" should we use it? might be worth adding a comment with link to polofill as some sort of doc

Reply:
Ah yes, I think that was some petulance after a long day! I believe this is a valid way of doing things, but very old-fashioned. I'll remove the comment since it serves no purpose.